V2/tezex/src/components/swap/style.tsx

```diff
@@ -0,0 +1,325 @@
+const classes = {
```

Comment:
I see we have 2 style files one with js and tsx extension are both needed? We should consolidate all styles into a single files

Reply:
yes styling is a bit of a mess, some things changed in mui for example useStyles does not work anymore it got depreciated ,I have not spent time at figuring a work around but i will do that this week and hopefully then I can consolidate the styles.